spec/fixtures/scorm_package/Module1/index.htm

```diff
@@ -0,0 +1,123 @@
+<!doctype html>
+<html>
+<link rel="stylesheet" type="text/css" href="../_sys/systemCheck.css" />

```

Comment:
We need the file to test whether or not additional files are being correctly added. I started with an actual entire scorm package, then it was slow so I removed most of it. I can remove the extraneous html though.